HARK/core.py

```diff
@@ -167,7 +273,16 @@ def assign_parameters(self, **kwds):
         for key in kwds:
             setattr(self, key, kwds[key])
 
+    assign_parameters_model = assign_parameters
+
     def get_parameter(self, name):
+        # CDC 20210527: A global search across all HARK, REMARKs,
+        # and DemARKs does not find any instance of this being used.
+        # We ought either to:
+        # 1. use it pervasively
+        #    * in which case there should be a companion set_parameter
+        # or
+        # 2. Eliminate it
```

Comment:
This is a good point.
I believe the issue is that the parameters dictionary was implemented for a few good reasons, but for backwards compatibility, it was made consistent with the 'whiteboard' way of accessing parameters through object attributes.